components/chips/chips.css

```diff
@@ -0,0 +1,21 @@
+@import url("../../styles/var.css");
+
+.chip-input {
```

Comment:
One of the feedback I received was to prefer the use of "rems" over "pixel". I see you've used px for border. I believe its a good practice to stick to rems here as well. The rest looks fine.